core/deployment/src/main/java/io/quarkus/deployment/util/ContainerRuntimeUtil.java

```diff
@@ -42,21 +43,20 @@ private ContainerRuntimeUtil() {
      * @return a fully resolved {@link ContainerRuntime} indicating if Docker or Podman is available and in rootless mode or not
      * @throws IllegalStateException if no container runtime was found to build the image
      */
-    public static ContainerRuntime detectContainerRuntime() {
-        return detectContainerRuntime(true);
-    }
-
-    public static ContainerRuntime detectContainerRuntime(List<ContainerRuntime> orderToCheckRuntimes) {
+    public static ContainerRuntime detectContainerRuntime(ContainerRuntime... orderToCheckRuntimes) {
```

Comment:
Due to this change, all callers of `detectContainerRuntime()` will fail with NoSuchMethodException unless they are recompiled. Let me send a PR to get the method back.